test/e2e/fixtures/Dockerfile.testenv

```diff
@@ -0,0 +1,3 @@
+FROM --platform="${TARGETPLATFORM}" alpine:latest
```

Comment:
I don't think we need to have this as a new separate image, we can reuse the other images / any image, and setup the environment variables and command in the pod config?

Reply:
So the point of this test case is that we found a bug which meant that environment variables where only taken from pod config and ignored from the container image, so this is a regression test to ensure this remains working

Reply:
Ah interesting, that could be clearer in the test naming

Reply:
For reference this was the bug if we want to link it in the commit: https://github.com/kata-containers/kata-containers/issues/5730

Reply:
That bugs describes something subtly different, where it is being defined in both places whereas from what I can tell this test is just setting it in the image?

Reply:
Yeah that is correct - I'd forgotten that the bug was that the merging of the config from the container and pod was done incorrectly and ignored the pod config, so I think we update the test to include that. Thanks for keeping me honest JT!